### PR TITLE
testing against python 3.4 (and 3.5)

### PR DESCRIPTION
**Pull request by brunobord**
refs #128

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ python: 2.7
 env:
     - TOX_ENV=py27
     - TOX_ENV=py33
+    - TOX_ENV=py34
+    - TOX_ENV=py35
```
Comment (Natim) [inline .travis.yml]: I doubt this will work we had to use a matrix in Kinto/kinto

Comment (brunobord) [inline .travis.yml]: it did work: https://travis-ci.org/novafloss/workalendar/builds/118364073

Comment (Natim) [inline .travis.yml]: That's a great news :)

Comment (brunobord) [inline .travis.yml]: you may want to see the result of ``python --version`` at the end of each test job: https://travis-ci.org/novafloss/workalendar/builds/118435186

Comment (Natim) [inline .travis.yml]: Yes I did :)

**Comment by jpic** [comment]
LGTM

**Pull request merged**